lib/api/v3/queries/query_representer.rb

```diff
@@ -185,6 +185,8 @@ def initialize(model,
 
         property :timeline_visible
 
+        property :show_hierarchies
```

Comment:
can't we simply use `hierarchies`? It is probably just my rails infected preference but I really dislike those two word properties for booleans. Same applies for `timeline_visible` for that matter.